src/test/java/uk/gov/register/service/ItemValidatorTest.java

```diff
@@ -151,4 +166,89 @@ public void validateItem_failsOnEmptyStringField() throws IOException {
+        JsonNode jsonNode = nodeOf(jsonString);
+
+        try {
+            countryItemValidator.validateItem(jsonNode, countryFieldsByName, countryRegisterMetadata);
+        } catch (ItemValidationException e) {
+            fail("Must not execute this statement");
+        }
+    }
+
+    @Test
+    public void validateItem_shouldValidateSuccessfully_whenInputDateTimeIsOfFormatYYYYDDMM() throws IOException {
+        String jsonString = "{\"country\":\"myCountry\",\"start-date\":\"2012-04\"}";
+        JsonNode jsonNode = nodeOf(jsonString);
+
+        try {
+            countryItemValidator.validateItem(jsonNode, countryFieldsByName, countryRegisterMetadata);
+        } catch (ItemValidationException e) {
+            fail("Must not execute this statement");
+        }
+    }
+
+    @Test
+    public void validateItem_shouldValidateSuccessfully_whenInputDateTimeIsOfFormatYYYYMMDD() throws IOException {
+        String jsonString = "{\"country\":\"myCountry\",\"start-date\":\"2012-04-01\"}";
+        JsonNode jsonNode = nodeOf(jsonString);
+
+        try {
+            countryItemValidator.validateItem(jsonNode, countryFieldsByName, countryRegisterMetadata);
+        } catch (ItemValidationException e) {
+            fail("Must not execute this statement");
+        }
+    }
+
+    @Test
+    public void validateItem_shouldValidateSuccessfully_whenInputDateTimeIsOfFormatYYYYMMDDThhmmss() throws IOException {
+        String jsonString = "{\"country\":\"myCountry\",\"start-date\":\"2012-04-01T23:23:23\"}";
+        JsonNode jsonNode = nodeOf(jsonString);
+
+        try {
+            countryItemValidator.validateItem(jsonNode, countryFieldsByName, countryRegisterMetadata);
+        } catch (ItemValidationException e) {
+            fail("Must not execute this statement");
+        }
+    }
+
+    @Test
+    public void validateItem_shouldValidateSuccessfully_whenInputDateTimeIsOfFormatYYYYMMDDThhmmssZ() throws IOException {
+        String jsonString = "{\"country\":\"myCountry\",\"start-date\":\"2012-04-01T23:23:23Z\"}";
+        JsonNode jsonNode = nodeOf(jsonString);
+
+        try {
+            countryItemValidator.validateItem(jsonNode, countryFieldsByName, countryRegisterMetadata);
+        } catch (ItemValidationException e) {
+            fail("Must not execute this statement");
+        }
+    }
+
+    @Test
+    public void validateItem_throwsValidationException_whenInputDateTimeIsNotValid() throws IOException {
+        String jsonString = "{\"country\":\"myCountry\",\"start-date\":\"18/12/2009\"}";
+        JsonNode jsonNode = nodeOf(jsonString);
+
+        try {
+            countryItemValidator.validateItem(jsonNode, countryFieldsByName, countryRegisterMetadata);
+            fail("Must not execute this statement");
+        } catch (ItemValidationException e) {
+            assertThat(e.getMessage(), equalTo("Field 'start-date' value must be of type 'datetime'"));
```

Comment:
I've remove it but, in my opinion, now the error message is misleading. It's complaining that it's not the correct data type, what can be true sometimes (Ex: "value12345') but it is saying the same when the value is a datetime but not in the correct format (Ex: 01/01/2018). What do you think @michaelabenyohai?

Reply:
I think this is fine because the the error says `value must be datetime` and by `datetime` we mean whatever is defined in https://datatype.register.gov.uk/record/datetime. In this case, it is true that both "value12345" and "01/01/2018" are not a `datetime` as per our register definition.

Reply:
I do believe that we need to better document these error messages though. This might be something to add to our Tech Docs. I'll speak to our Tech Writer and see.